src/commands/send.rs

```diff
@@ -169,6 +175,126 @@ fn print_response(blob: Vec<u8>, method_name: String) -> AnyhowResult {
             let response = Decode!(blob.as_slice(), ManageNeuronResponse)?;
             println!("Response: {:?\n}", response);
         }
+        SupportedResponse::SnsCanistersSummary => {
+            let response: Vec<(String, PrincipalId, CanisterStatusResultV2)> = Decode!(
+                blob.as_slice(),
+                Vec<(String, PrincipalId, CanisterStatusResultV2)>
+            )?;
+            for (name, principal, canister_status) in response.into_iter() {
+                println!("\nCanister Summary:");
+                println!("name: {}", name);
+                println!("id: {}", principal);
+                println!("controller: {}", canister_status.controller());
+                println!("status: {}", canister_status.status());
+                println!(
+                    "module hash: {}",
+                    &canister_status
+                        .module_hash()
+                        .map_or("None".to_string(), |hash| hex::encode(hash))
+                );
+                println!("memory size: {}", canister_status.memory_size());
+                println!("cycles: {}", canister_status.cycles());
+                println!(
+                    "freezing threshold: {}",
+                    canister_status.freezing_threshold()
+                );
+            }
+        }
+        SupportedResponse::NervousSystemParameters => {
+            let response = Decode!(blob.as_slice(), NervousSystemParameters)?;
+            println!("\nNervous System Parameters:");
+            println!(
+                "reject cost e8: {}",
+                response
+                    .reject_cost_e8s
+                    .map_or("None".to_string(), |field| field.to_string())
+            );
+            println!(
+                "transaction fee e8: {}",
+                response
+                    .transaction_fee_e8s
+                    .map_or("None".to_string(), |field| field.to_string())
+            );
+            println!(
+                "max proposals to keep per action: {}",
+                response
+                    .max_proposals_to_keep_per_action
+                    .map_or("None".to_string(), |field| field.to_string())
+            );
+            println!(
+                "initial voting period: {}",
+                response
+                    .initial_voting_period
+                    .map_or("None".to_string(), |field| field.to_string())
+            );
+            // TODO(alejandro): improve the display of default followees
```

Comment:
Did you mean to do this in this PR or will you do it in a followup? If in a followup can you attach a JIRA to this comment as well?

Reply:
I created [this](https://dfinity.atlassian.net/browse/NNS1-1395) issue to track it.